src/bot/commands/delete.ts

```diff
@@ -0,0 +1,110 @@
+import Command from '../lib/command';
+import {
+  APIEmbed,
+  ApplicationCommandOptionType,
+  ApplicationCommandType,
+  ButtonStyle,
+  ChatInputCommandInteraction,
+  ComponentType,
+  Message,
+  roleMention
+} from 'discord.js';
+import { buildJokeDisplay, interactionInfo, interactionValidate, waitForInteraction } from '../utils';
+import { CategoriesRefs } from '../../typings';
+import { jokeById, jokeByQuestion } from '../../controllers';
+import { Colors } from '../constants';
+import Jokes from '../../jokes';
+import prisma from '../../prisma';
+
+export default class DeleteCommand extends Command {
+  constructor() {
+    super({
+      name: 'delete',
+      description: 'Supprimer une blague',
+      type: ApplicationCommandType.ChatInput,
+      options: [
+        {
+          type: ApplicationCommandOptionType.String,
+          name: 'recherche',
+          description: 'ID ou question de la blague',
+          required: true
+        }
+      ]
+    });
+  }
+
+  async run(interaction: ChatInputCommandInteraction<'cached'>) {
+    if (interaction.user.id !== '207190782673813504') {
```

Comment:
Oublie pas c'est de l'open source donc pas de userID comme ca :)

```suggestion
   if (interaction.user.id !== process.env.BOT_USER_OWNER) {
```